src/igvfd/types/file_set.py

```diff
@@ -1584,7 +1578,7 @@ class ConstructLibrarySet(FileSet):
         'notSubmittable': True
     })
     def applied_to_samples(self, request, applied_to_samples):
-        return paths_filtered_by_status(request, applied_to_samples)
+        return paths_filtered_by_status(request, applied_to_samples) or None
```

Comment:
Could be useful to keep as returning an empty array as all construct library sets should eventually be applied to a sample and displaying an empty array could be informative.